src/screens/wallet-overview/components/wallet-overview-shimmer.component.tsx

```diff
@@ -43,7 +48,7 @@ export const WalletOverviewShimmer = () => {
             {Actions.map((items) => (
               <Box hCenter vCenter gap={1} key={items.id}>
                 <Square />
-                <Text variant='small' text={items.title} />
+                <SmallLine />
```

Comment:
Replace this text `<Text variant='small' text='Activities' color='textLight' bold600 />` with a rectangle.

Reply:
done. replaced with rectange.